### PR TITLE
Various minor typo fixes

### PR DESCRIPTION
**Pull request by biodranik**

### CODE REVIEW

File: plugins/janus_videoroom.c [hunk 1]
```diff
@@ -87,7 +87,7 @@ room-<unique room ID>: {
 	notify_joining = true|false (optional, whether to notify all participants when a new
 				participant joins the room. The Videoroom plugin by design only notifies
 				new feeds (publishers), and enabling this may result extra notification
-				traffic. This flag is particularly useful when enabled with \c require_pvtid
+				traffic. This flag is particularly useful when enabled with require_pvtid
```
Comment (lminiero) [inline plugins/janus_videoroom.c:90]: Why did you remove the `\c` here?

File: plugins/janus_videoroom.c [hunk 2]
```diff
@@ -554,7 +554,7 @@ room-<unique room ID>: {
  * the response to a \c joinandconfigure will be a \c joined event, and
  * will again be accompanied by a JSEP SDP answer as usual.
  *
- * However you decided to publish something, as soon as the PeerConnection
+ * However if you decided to publish something, as soon as the PeerConnection
```
Comment (lminiero) [inline plugins/janus_videoroom.c:557]: The "if" is wrong here: what I meant was "no matter how you decided to publish something" (since we just talked of different ways of doing that).

File: plugins/janus_videoroom.c [hunk 3]
```diff
@@ -408,7 +408,7 @@ room-<unique room ID>: {
  * (although may choose not to, more on this later) publish media in the
  * room, and as such become feeds that you can subscribe to.
  *
- * To specify a handle will be associated with a publisher, you must use
+ * To specify a handle which will be associated with a publisher, you must use
```
Comment (lminiero) [inline plugins/janus_videoroom.c:411]: The "which" is wrong here: the sentence means to say that "to specify THAT a (core) handle will be associated (is going to be associated) with a publisher, you mus use...".

File: plugins/janus_videoroom.c [hunk 4]
```diff
@@ -895,7 +895,7 @@ room-<unique room ID>: {
  * right info out of band (which is impossible in rooms configured with
  * \c require_pvtid).
  *
- * To specify a handle will be associated with a subscriber, you must use
+ * To specify a handle which will be associated with a subscriber, you must use
```
Comment (lminiero) [inline plugins/janus_videoroom.c:898]: See above.

File: plugins/janus_videoroom.c [hunk 5]
```diff
@@ -933,7 +933,7 @@ room-<unique room ID>: {
  * or are not interested in getting all the media (e.g., they're ok with
  * just audio and not video, or don't have enough bandwidth for both),
  * they can use those properties to shape the SDP offer to their needs.
- * In case the publisher to subscribe to is simulcasting or doing VP9 SVC,
+ * In case the publisher is simulcasting or doing VP9 SVC,
```
Comment (lminiero) [inline plugins/janus_videoroom.c:936]: Why did you remove that?

**Review by lminiero — COMMENTED** [review]
Apologies for the late response, we were on vacation. While there are some nice fixes on typos I wasn't aware of, there are some changes that are broken. I added them as notes below.

File: plugins/janus_videoroom.c (hunk 1 above)
Comment (biodranik) [reply plugins/janus_videoroom.c:90]: Because this text is verbatim already, and \c [prints like "\c require_pvtid"](https://janus.conf.meetecho.com/docs/videoroom.html)

**Comment by biodranik** [comment]
@lminiero Thanks for the review, I have reverted back changes you've mentioned.

File: plugins/janus_videoroom.c (hunk 1 above)
Comment (lminiero) [reply plugins/janus_videoroom.c:90]: Oh right, good catch!
